services/core-web/src/components/common/wrappers/ScrollContentWrapper.js

```diff
@@ -10,30 +10,40 @@ import { PropTypes } from "prop-types";
 const propTypes = {
   id: PropTypes.string.isRequired,
   children: PropTypes.element.isRequired,
-  location: PropTypes.shape({ hash: PropTypes.string }).isRequired,
   title: PropTypes.string.isRequired,
+  history: PropTypes.shape({
+    location: PropTypes.shape({ state: PropTypes.shape({ currentActiveLink: PropTypes.string }) }),
+  }).isRequired,
 };
 
-export const ScrollContentWrapper = (props) => {
-  const isActive = () =>
-    includes(props.location.hash, props.id) ? "circle purple" : "circle grey";
+class ScrollContentWrapper extends Component {
+  isActive = () => {
```

Comment:
Looks like this doesn't use state, can you convert back to a stateless component.

Reply:
Done!